src/redisraft.h

```diff
@@ -580,6 +607,18 @@ typedef struct RaftReq {
             int fail;
             int delay;
         } debug;
+        ImportKeys import_keys;
+        struct {
+            char shardGroupId[RAFT_DBID_LEN+1];
```

Comment:
```suggestion
            char shard_group_id[RAFT_DBID_LEN+1];
```
Please use snake case for variable names.